server/text_generation_server/models/cache_manager.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         ]
         self.free_block_mask = torch.ones(num_blocks, dtype=torch.int32, device="cpu")
         self.slots = torch.arange(
-            0, num_blocks * self.block_size, dtype=torch.int32
+            0, num_blocks * self.block_size, dtype=torch.int64
```

Comment:
quick question, is there a case where `num_blocks` is really really big? just trying to understand the type change

Reply:
It's because the vllm kernel now ask for this dtype. I don't know why they changed. 
Slots is a very small tensor anyway.